### PR TITLE
[Grid] [Asset] Save Configuration should return configuration schema

### LINKED ISSUE #484: [Grid] [Asset] Save Configuration should return id
When saving a new grid configuration the response should include the saved object with id.

### PR DESCRIPTION
**Pull request by martineiber**
## Changes in this pull request
Resolves #484

### CODE REVIEW

File: src/Asset/Controller/Grid/Configuration/SaveConfigurationController.php [hunk 1]
```diff
@@ -72,8 +75,8 @@ public function __construct(
     public function saveAssetGridConfiguration(
         #[MapRequestPayload] SaveConfigurationParameter $saveConfigurationParameter
     ): Response {
-        $this->gridSaveConfigurationService->saveAssetGridConfiguration($saveConfigurationParameter);
+        $configuration = $this->gridSaveConfigurationService->saveAssetGridConfiguration($saveConfigurationParameter);
```
Comment (mattamon) [inline src/Asset/Controller/Grid/Configuration/SaveConfigurationController.php:78]: Title and return does not match. Title says return id, but the save now returns the full configuration right?

Comment (martineiber) [reply src/Asset/Controller/Grid/Configuration/SaveConfigurationController.php:78]: Yeah i was lazy to add more description :D
The main point was that @vin0401  needs an ID. We decided we will provide the same object as used in the listing of configurations. 

This is not the Full configuration this is a reduced version with ID, Name and Description

Comment (mattamon) [reply src/Asset/Controller/Grid/Configuration/SaveConfigurationController.php:78]: Okay :D then please just update the title for the PR, so there is no confusion in the commit history :D

Comment (martineiber) [reply src/Asset/Controller/Grid/Configuration/SaveConfigurationController.php:78]: JFYI: The full Configuration is under the DetailedConfiguration schema

**Review by mattamon — APPROVED** [review]
It got the looks.... of good!
🦊

**Pull request merged**